includes/class-wcmp-postcode-fields.php

```diff
@@ -67,6 +67,8 @@ public function initialize(): void
             add_filter('woocommerce_load_order_data', [$this, 'load_order_data']);
 
             // Custom shop_order details.
+            add_filter('woocommerce_admin_billing_fields', [$this, 'addSeparateAdminAddressFields']);
```

Comment:
We noemen dit op andere plekken "split" address fields, misschien is het beter om die terminologie hier ook te gebruiken

Reply:
Ik vind zelf dat "split address fields" ook geen goede benaming is. Maar ik zal hem aanpassen.